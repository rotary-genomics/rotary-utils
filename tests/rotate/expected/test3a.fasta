>sequence1
CCCCTTTTTGGGGGAAAAAC
>sequence2 # comment
CCCAAAGGGTTT
>sequence3
AATTCCGG
