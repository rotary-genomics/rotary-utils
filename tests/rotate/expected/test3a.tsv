sequence_id	sequence_length_bp	rotate_type	rotate_value
sequence1	20	fraction	0.3
sequence2	12	fraction	0.3
sequence3	8	fraction	0.3
